package.json

```diff
@@ -25,5 +25,12 @@
     "module": "tslib.es6.js",
     "jsnext:main": "tslib.es6.js",
     "typings": "tslib.d.ts",
-    "sideEffects": false
+    "sideEffects": false,
+    "exports": {
+        ".": {
```

Comment:
Please provide `"module"` condition as well. It's going to be supported by the upcoming webpack 5 and will allow it to keep only a single copy of `tslib` in the bundle. Point of reference: https://gist.github.com/sokra/e032a0f17c1721c71cfced6f14516c62#reference-syntax

Reply:
Sure, can be added afterwards.

There are no real docs for this yet on their page but this gist has been updated multiple times when they have been iterating over semantics. This is already implemented and shipped in webpack 5 RC.

Reply:
@Andarist do you have any examples of packages using the `"module"` condition successfully for reference here?

Reply:
@guybedford I did not (mainly because conditional exports themselves are rather rare to find so far), but I've decided to showcase this using one of my smaller modules. Here are exports that I have defined:
https://github.com/Andarist/callbag-last-element/blob/07099fd55b4dfff80a7697065c71ce9b5fc07ade/package.json#L7-L14
and here is the webpack output:
https://github.com/Andarist/webpack-module-condition-test/tree/7bca3a5ae4f07b5429c7b36db3d43b2072b39cf2/src
Notice how `last` function is only there a single time in this bundle even when mixing ESM and CJS files.

The input for this is as simple as:
```js
// cjs-file.js
module.exports = require("callbag-last-element").default;

// index.js
import lastElement from "callbag-last-element";
import lastElementFromCjs from "./cjs-file";

console.log({ lastElement, lastElementFromCjs });
```
and can be found [here](https://github.com/Andarist/webpack-module-condition-test/tree/7bca3a5ae4f07b5429c7b36db3d43b2072b39cf2/src) (same repo)